docs/getting-started-with-snowplow-bdp/cloud/naming-resources-in-databricks/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Naming resources in Databricks"
+sidebar_position: 3
+---
+
+# Naming resources in Databricks
+
+:::caution
+
+We advise to use the default resource names in the setup script provided in Console. If you need to follow your own naming convention however, please follow the guidelines below and provide Snowplow with the resource names you used through the Databricks setup in BDP Console.
+
+:::
+
+### `HTTP path`
+
+[Databricks compute resources URL](https://docs.databricks.com/integrations/jdbc-odbc-bi.html#building-the-connection-url-for-the-databricks-driver).
+
+It should start with a slash character.
```

Comment:
this is fixed now, we do accept HTTP paths not starting with `/`

Reply:
Fixed, thx!